vcd/resource_vcd_nsxt_security_group_test.go

```diff
@@ -447,10 +463,8 @@ resource "vcd_network_isolated_v2" "nsxt-backed" {
 }
 resource "vcd_nsxt_security_group" "group1" {
   org = "{{.Org}}"
-  vdc = "{{.NsxtVdc}}"
```

Comment:
Seeing that `vdc` field has been removed from all the tests above, do you still test backwards compatibility? If yes, where?

Reply:
There is huge test: TestAccVcdNsxtSecurityGroupInheritedVdc